jcl/src/java.base/share/classes/java/lang/Access.java

```diff
@@ -499,9 +499,15 @@ public String join(String prefix, String suffix, String delimiter, String[] elem
 /*[IF JAVA_SPEC_VERSION >= 20]*/
 	@Override
 /*[IF JAVA_SPEC_VERSION >= 22]*/
+/*[IF JAVA_SPEC_VERSION >= 24]*/
+	public void ensureNativeAccess(Module mod, Class<?> clsOwner, String methodName, Class<?> clsCaller, boolean jni) {
+		mod.ensureNativeAccess(clsOwner, methodName, clsCaller, jni);
+	}
```

Comment:
Suggest the parameter names could be improved, and with three variants, I think this should use `ELSEIF`:
```java
/*[IF JAVA_SPEC_VERSION >= 24]*/
	public void ensureNativeAccess(Module module, Class<?> ownerClass, String methodName, Class<?> callerClass, boolean isJNI) {
		module.ensureNativeAccess(ownerClass, methodName, callerClass, isJNI);
	}
/*[ELSEIF JAVA_SPEC_VERSION >= 22]*/
	public void ensureNativeAccess(Module module, Class<?> ownerClass, String methodName, Class<?> callerClass) {
		module.ensureNativeAccess(ownerClass, methodName, callerClass);
	}
/*[ELSEIF JAVA_SPEC_VERSION >= 20]*/
	public void ensureNativeAccess(Module module, Class<?> ownerClass, String methodName) {
		module.ensureNativeAccess(ownerClass, methodName);
	}
/*[ENDIF] JAVA_SPEC_VERSION >= 24 */
```

Reply:
Updated with `ELSEIF` JPP tag.